### DISCUSSION TITLE
Can't figure out how to unbind `<C-l>`

### DISCUSSION THREAD

**Discussion by mfdorst** [LunarVim/LunarVim#3438] [Q&A]
When using a floating terminal (toggleterm), if I do `Ctrl+L` to clear the screen it closes the terminal instead. This is very irritating for me, since I do that constantly when using a terminal. I've been trying to unbind `<C-l>` with no success.

I have tried:
```
lvim.lsp.buffer_mappings.normal_mode["<C-l>"] = nil
lvim.lsp.buffer_mappings.normal_mode["<C-L>"] = nil
lvim.keys.normal_mode["<C-l>"] = nil
lvim.keys.normal_mode["<C-L>"] = nil
```
but none of those work. When I do `:verbose map <C-l>` it shows this:
```
n  <C-L>       * <C-W>l                                                                                                                                                                      
        Last set from Lua
```
I'm pretty sure this isn't the culprit, it should be `<C-l>` not `<C-L>`, but even after trying to unbind both, as shown above, this still persists.

Does anyone know what I can do to get `Ctrl+L` to work in the floating terminal?

**Comment by izzy-el** [comment]
Hey there! You can try and delete `<C-l>` in Terminal Mode. Add this to your configuration file:
`vim.keymap.del("t", "<C-l>")`

You can find more information on [this issue](https://github.com/akinsho/toggleterm.nvim/issues/227).
